include_radius: 0.10
exclude_radius: 0.50
min_points: 3
distance_filter: 3.0
